
ldap3.utils.tls_backportg
CertificateError)ldap3.utils.tls_backport.CertificateError"builtins.ValueErrorj36j37j38j39j310f
match_hostname'ldap3.utils.tls_backport.match_hostname*
cert*
hostnamez36z37z38z39z310*©
__annotations__(ldap3.utils.tls_backport.__annotations__W
builtins.dict[builtins.str,Any]
builtins.str"builtins.str
Any"builtins.dict*36*37*38*39*310